modules/table/src/main/java/org/apache/ignite/internal/table/distributed/StorageUpdateHandler.java

```diff
@@ -118,8 +149,29 @@ public void handleUpdateAll(
             Map<UUID, ByteBuffer> rowsToUpdate,
             TablePartitionId commitPartitionId,
             @Nullable Consumer<Collection<RowId>> onReplication
+    ) {
+        handleUpdateAll(txId, rowsToUpdate, commitPartitionId, onReplication, null);
+    }
+
+    /**
+     * Handle multiple updates.
+     *
+     * @param txId Transaction id.
+     * @param rowsToUpdate Collection of rows to update.
+     * @param commitPartitionId Commit partition id.
+     * @param onReplication On replication callback.
+     * @param lowWatermark GC low watermark.
+     */
+    public void handleUpdateAll(
+            UUID txId,
+            Map<UUID, ByteBuffer> rowsToUpdate,
+            TablePartitionId commitPartitionId,
+            @Nullable Consumer<Collection<RowId>> onReplication,
+            @Nullable HybridTimestamp lowWatermark
```

Comment:
You need to create a ticket (and add TODO) to support this argument with write commands or something like that.